extractor/build.gradle

```diff
@@ -8,4 +8,5 @@ dependencies {
     implementation 'org.nibor.autolink:autolink:0.10.0'
 
     testImplementation 'junit:junit:4.13'
+    testImplementation "com.squareup.okhttp3:okhttp:3.12.11"
```

Comment:
Can't you use the latest version? Since this only needs to be run in tests, it does not need to be API19-compatible

Reply:
Yes, however I think it's a good idea to use the same version in NewPipe and NewPipeExtractor's tests. OkHttp 3.12 will also just still be supported until December 31, 2021.

Reply:
Ok, I see